.internal-ci/helm/watcher/templates/watcher-backup-scripts.yaml

```diff
@@ -49,12 +52,29 @@ data:
     # Set sha256 signing
     ${AWS} configure set default payload_signing_enabled=true
 
+    # Create the copy directory
+    mkdir -p /watcher-backup
+
     # Copy the db file for stable upload
-    cp /watcher/data.mdb /tmp/data.mdb
+    mdb_copy /watcher /watcher-backup
+
+    cd /watcher-backup
+    $MDB_STAT -s watcher_db:block_signatures . 2> /dev/null
```

Comment:
Whats the reason you reference `mdb_stat` with a variable and not `mdb_copy`?   I think either is fine, but we should be consistent.